docs/builtins.md

```diff
@@ -0,0 +1,718 @@
+# Builtins
+
+Evy provides built-in functions and events that allow for user
+interaction, graphics, animation, mathematical operations, and more.
+
+Functions are self-contained blocks of code that perform a specific
+task. Events are notifications that are sent to a program when
+something happens, such as when a user moves the mouse or presses a
+key.
+
+## Input and Output
+
+### `print`
+
+`print` prints the arguments given to it to the output area. It separates them by
+a single space and outputs a newline character at the end.
+
+#### Example 
+
+```evy
+print "Hello"
+print 2 true "blue"
+print "array:" [1 2 3]
+print "map:" {name: "Scholl" age: 21}
+```
+Output
+```
+Hello
+2 true blue 
+array: [1 2 3]
+map: {name:Scholl age:21}
+```
+
+#### Reference 
+
+    print a:any...
+
+`print` prints its arguments to the output area, each separated by a
+single space and terminated by a newline character. If no arguments are
+provided it only prints the newline character. 
+
+A tab is printed for the `\t` control character. A newline is printed
+for the `\n` control character. Quotes in string literals are escaped
+with backslash `\`, for example `print "Here's a tab: \t.\nShe said: \"Thank you!\""`.
+
+In a browser environment `print` typically prints to a DOM Element wired
+to the evy runtime. When using evy as Command Line Interface, `print`
+prints to Stdout.
+
+### `read`
+ 
+`read` reads a line of input from the user and returns it as a
+string. The newline character is not included in the returned string.
+
+#### Example
+```evy
+name := read
+print "Hello, "+name+"!"
+```
+
+Input
+```
+Mary Jackson
+```
+
+Output
+```
+Hello, Mary Jackson!
+```
+
+#### Reference 
+
+    read:string
+
+`read` returns a string that contains the line of input that the user
+entered up until, excluding the newline character. It is a blocking
+functions, which means that it will not return until the user has
+entered a line of input and pressed the Enter key.
+
+In a browser environment `read` typically reads from a DOM Textarea
+Element wired to the evy runtime. When using evy as Command Line
+Interface, `read` reads from Stdin.
+
+### `cls`
+
+`cls` clears the output area of all printed text.
+
+#### Example
+```evy
+print "Hello"
+sleep 1
+cls
+print "Bye"
+```
+
+Output
+```
+Bye
+```
+
+#### Reference 
+
+    cls
+
+In a browser environment `cls` typically clears the DOM Element wired to
+the evy runtime which is used for printing. When using evy as Command
+Line Interface, `cls` clears the terminal, similar to the Unix `clear`
+or Windows `cls` commands.
+
+### `printf`
+
+`printf` stands for print formatted.
+
+`printf` prints its arguments to the output area according to a _format_
+string. The format string is the first argument, and it
+contains _specifiers_. Specifiers are placeholder that start with a
```

Comment:
I think you could remove "are placeholders that", so it reads: Specifiers start with a ...

Introducing the word "placeholder" when not used elsewhere doesn't really add anything here I think.

Reply:
ok, agreed, thank you!